src/asmcnc/apps/drywall_cutter_app/material_setup_popup.py

```diff
@@ -305,6 +306,8 @@ def __init__(self, localization, keyboard, dwt_config, **kwargs):
         for text_input in self.text_inputs:
             text_input.bind(focus=self.text_on_focus)
         self.depth_per_pass.bind(text=self.warning_pass_depth)
+        self.cut_depth_warning.bind(parent=Clock.schedule_once(self.disable_confirm_button))
+        self.pass_depth_warning.bind(parent=Clock.schedule_once(self.disable_confirm_button))
```

Comment:
Do this inside of a function so that you can unbind it easily. 

def ...
    Clock.schedule_once